src/main/java/com/bugsnag/Session.java

```diff
@@ -0,0 +1,50 @@
+package com.bugsnag;
+
+import com.bugsnag.serialization.Expose;
+
+import java.util.Date;
+import java.util.concurrent.atomic.AtomicInteger;
+
+class Session {
+
+    private final String id;
+    private final Date startedAt;
+    private final AtomicInteger handledCount;
+    private final AtomicInteger unhandledCount;
+
+    Session(String id, Date startedAt) {
+        this.id = id;
+        this.startedAt = new Date(startedAt.getTime());
```

Comment:
Unfortunately `Date` in Java is mutable. I've made a defensive copy here and in the getter to prevent anyone from altering the internals of the `Session` class.